api/src/main/resources/liquibase.xml

```diff
@@ -8,6 +8,12 @@
         See http://www.liquibase.org/manual/home#available_database_refactorings
         for a list of supported elements and attributes
     -->
+	<changeSet id="ugandaemr-07032017-1958" author="ssmusoke">
+		<comment>Delete coded obs with null values from the summary page - most of these seem to be health education related concepts which are not on the summary page</comment>
+		<sql>
+			DELETE FROM obs WHERE concept_id IN (99169, 99170, 99171,99172, 99173,99174, 99175,99176,99177,99178,99179,99180,99181) AND value_coded IS NULL
```

Comment:
@dkayiwa I have no idea how these got entered, however this database was migrated so I suspect that may have happened in the 1.6.x versions of the forms. Its the first time that I am seeing this error and took about half a day to chase down. 

We are finding that the validation checks in the 2.x migration are more stringent than during 1.x - so I expect to see a lot of these happening